### ISSUE TITLE
MSI installer: vjoy is not currently running

### ISSUE THREAD

**Issue by bofhgr** [WhiteMagic/JoystickGremlin#73]
![image](https://user-images.githubusercontent.com/28956359/33811371-462a84be-de1b-11e7-9827-f23569738bd3.png)

Hi, first time user here, apologies if this is not an issue and I failed to follow installation instructions properly.

I downloaded and run the Joystick Gremlin msi installation. Upon starting the Joystick Gremlin from the desktop I get this error message. Do I need to download vJoy separately?

**Comment by WhiteMagic** [comment]
Yeah at the moment Gremlin doesn't package the vJoy installer and you have to make sure you install it yourself before using Gremlin. It's mentioned in the manual here https://whitemagic.github.io/JoystickGremlin/overview/ but it's not ideal since nothing actually even informs people of the existence of that page / manual :-)

For now you'll have to install vJoy yourself using the links on the page linked above. Short term I probably should add links to the manual and a reminder that vJoy has to be installed to the error you're getting when it's not running. Packaging the vJoy installer with the Gremlin installer is certainly doable but all these installers are a nightmare so I'm kind of keep putting it off to not have to suffer right now but only a bit later :-)
